item-impl/src/main/java/com/example/auction/item/impl/PItemCommand.java

```diff
@@ -43,6 +41,92 @@ public int hashCode() {
         }
     }
 
+
+    final class UpdateItem implements PItemCommand, PersistentEntity.ReplyType<PUpdateItemResult> {
+        private final UUID id;
+        private final UUID creator;
```

Comment:
We don't. I just needed those two fields on readside and added them on the event. Then... intertia kicked in and added them on the command too. Fixing. Thanks.